### DISCUSSION TITLE
Eeabling Grid overlay

### DISCUSSION THREAD

**Discussion by federicopeyrani1969** [orbat-mapper/orbat-mapper#159] [General]
Hi Kjell,

is it possible to visualize over the map the grid reference with eastings and northings?
And maybe enabling the search through grid coordinates (6 numbers).
It would be great to place on the map data from Allied sources!

Regards
Federico

**Comment by kjellmf** [comment]
Do you mean MGRS grid or UTM grid?

**Reply by kjellmf** [reply]
Found a few examples:
- https://viglino.github.io/ol-ext/examples/canvas/map.control.graticule2.html
- https://viglino.github.io/ol-ext/examples/canvas/map.control.graticule.html

**Reply by federicopeyrani1969** [reply]
I mean MGRS.
thank you
Federico

**Reply by federicopeyrani1969** [reply]
To explain me better, I would like to be able to find in the quickest time the universal coordinates I find in the allied reports.
for example, in my text about the 45th US ID I find:
"Commanding General and G-2, G-3 and G-4 landed on beach Green at 0830. CP established at 367159."
I would like to find this **367159** at once on the map. I am not interested in the degrees, but in eastigns and northings, like the ones displayed at right in this attached map:
![20140714_155554](https://github.com/orbat-mapper/orbat-mapper/assets/58425768/4f5cb7eb-64da-454e-9fc7-fb4b45c5ea8a)
Regards
Federico

**Reply by federicopeyrani1969** [reply]
or those ones in this map of Tunisia:
![centre map](https://github.com/orbat-mapper/orbat-mapper/assets/58425768/c81b05f3-0810-4eb6-af87-834f5c8b4e99)

**Reply by kjellmf** [reply]
Nice maps! For this to work you will also need the grid zone prefix. Do you know the grid zone for the 367159 example?

**Reply by kjellmf** [reply]
Did they actually used MGRS during WWII? I see the Cassini grid mentioned in some places.

**Reply by federicopeyrani1969** [reply]
Hi Kjell, 
I confirm every allied units (land sea and air) used the same reference grid for missions and reports. Sorry but I don't know grid zone for Sicily (the letters before the numbers, are they?). I think I saw once a reference map for grid zones, but think it's lost in my HD now. Allied reports almost always cite grid coordinates and not grid zone, rarely main cities instead of coords.
Sorry but I don't know map theory, so Cassini is something obscure to me. I didn't even know it was related to grids, I thought it was related to map representation. I can't help with your second question.
Best Regards 
Federico

**Reply by federicopeyrani1969** [reply]
Hi there, I'm back! How about this? Please let me kow if there are any changes.

**Reply by kjellmf** [reply]
I have not done any work on this yet (#160). Lots of math to figure out.
